src/internal/connector/exchange/service_restore.go

```diff
@@ -651,19 +651,27 @@ func establishContactsRestoreLocation(
 
 	ctx = clues.Add(ctx, "is_new_cache", isNewCache)
 
-	temp, err := ac.Contacts().CreateContactFolder(ctx, user, folders[0])
-	if err != nil {
+	fold, err := ac.Contacts().CreateContactFolder(ctx, user, folders[0])
```

Comment:
email and events coming in a follow-up PR?

Reply:
yeah! just wanted to keep this PR small and focused.